src/cysignals/signals.pyx

```diff
@@ -193,22 +205,23 @@ def init_cysignals():
     import signal
     old = signal.signal(signal.SIGINT, python_check_interrupt)
 
-    setup_alt_stack()
+    IF UNAME_SYSNAME != 'Windows':
+        setup_alt_stack()
```

Comment:
No need for that.

Reply:
This one cause an `undefined reference to 'setup_alt_stack'` if removed

Reply:
> This one cause an undefined reference to 'setup_alt_stack' if removed

Yes of course. But then you just implement `setup_alt_stack` to do nothing on Windows. This is related to the "major comment" that you should try to implement things as much as possible the same for POSIX and Windows.

Reply:
Ok i get the idea. Thx.